### ISSUE TITLE
problem: currency comparison typeahead is broken

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#996]
Problem: currency comparison typeahead is broken.

**Comment by chamidu-supeshala** [comment]
fixed in #999

**Comment by janat08** [comment]
that pull request will break functionality elsewhere.

**Comment by janat08** [comment]
adjust this line: https://github.com/chamidu-supeshala/blockrazor/blob/5e119b93ce70ed1c2b53de21e9511bc4f774f29b/imports/ui/components/typeahead.js#L66

**Comment by chamidu-supeshala** [comment]
Hi, @janat08 thanks for your suggestion. I have tested but no issue found after my fix. However as you suggested i will cancel my PR, and will adjust there as you mentioned.

**Comment by janat08** [comment]
This is a reusable template, it's used elsewhere.

**Comment by janat08** [comment]
btw: createElement is dead code.

**Comment by gazhayes** [comment]
@janat08 I haven't looked at the code, but it should be **_very_** clear in the code that this is a reusable template used elsewhere, it should be very well commented. If not, it will definitely be broken by someone. In fact, reusable stuff should really have associated tests, I haven't looked into client side tests though (and something like snapshot is not robust enough for this).

**Comment by janat08** [comment]
there're couple of pargraphs describing params and usage.

**Comment by janat08** [comment]
along with actual usages

**Comment by gazhayes** [comment]
OK. We definitely need tests then. If you didn't see this in time I would have merged it. It would be ok (the C4 takes care of it eventually) but it would still have been very annoying.

**Comment by janat08** [comment]
yes, that's good, you're creating a default template for when a template with event for adding currently typed value wasn't provided to the template.

**Comment by chamidu-supeshala** [comment]
Now i got it :)

**Comment by chamidu-supeshala** [comment]
fixed in #1004
